### PR TITLE
save trading fees and volume per day

### PR DESCRIPTION
**Pull request by paco0x**

### CODE REVIEW

File: src/mappings/clearingHouse.ts [hunk 1]
```diff
@@ -136,6 +137,11 @@ export function handlePositionChanged(event: PositionChangedEvent): void {
     protocol.blockNumber = event.block.number
     protocol.timestamp = event.block.timestamp
 
+    // update ProtocolDayData
+    const protocolDayData = getOrCreateProtocolDayData(event)
+    protocolDayData.tradingFee = protocolDayData.tradingFee.plus(positionChanged.fee)
+    protocolDayData.tradingVolume = protocolDayData.tradingVolume.plus(abs(positionChanged.exchangedPositionNotional))
```
Comment (vinta) [inline src/mappings/clearingHouse.ts:143]: Should we also do this in `handlePositionClosed`?

**Review by vinta — APPROVED** [review]

File: src/mappings/clearingHouse.ts (hunk 1 above)
Comment (paco0x) [reply src/mappings/clearingHouse.ts:148]: updated

**Comment by vinta** [comment]
@paco0x Sorry, my bad. Could you rebase this PR on the current `develop` since new branch should be based on `develop`?

**Comment by paco0x** [comment]
> @paco0x Sorry, my bad. Could you rebase this PR on the current `develop` since new branch should be based on `develop`?

Sure, updated

**Pull request merged**